src/css/custom.css

```
@@ -419,14 +439,53 @@ div.markdown h3 a:hover {
 
 .mediaImage {
   width: 100%;
-  height: auto
+  height: auto;
 }
 
 /* remove extra vertical space from tabs when it contains a code block */
-.tabs-container .theme-code-block {
-  margin-top: -1.4rem;
+.tabs-container .margin-top--md .theme-code-block {
+  margin-top: -1rem;
+}
+
+/* code title styling */
+div.codeBlockTitle_node_modules-\@docusaurus-theme-classic-lib-theme-CodeBlock-Content-styles-module {
+  font-size: 1.05rem;
+  font-weight: 700;
+  color: var(--ifm-color-gray-300);
+  border-bottom: 3px solid var(--ifm-code-background-color-light);
```

This selector won't hold up, it's targeting the theme you have in your node modules.  Notice they didn't get applied in the preview URL. I did this a lot in the beginning 😄 

Some dynamic info gets added to these after it builds so you should be fine to target this block like this. Also tweaked the border-bottom to be a little more subtle. 

```suggestion
div[class^='codeBlockTitle'] {
  font-size: 1.05rem;
  font-weight: 700;
  color: var(--ifm-color-gray-300);
  border-bottom: 2px solid var(--ifm-code-background-color-light);
}
```